package.json

```diff
@@ -2,7 +2,7 @@
   "name": "pivotal-swimlanes",
   "description": "Swimlane view for current Pivotal sprint",
   "engines": {
-    "node": "4.0.0",
+    "node": "6.5.0",
```

Comment:
I agree with this, could you put it in a separate commit though? Just for clarity if going through the commit log